spec/enumerable_spec.rb

```diff
@@ -0,0 +1,168 @@
+require '../enumerables.rb'
+
+describe Enumerable do
+  test_array1 = [11, 2, 3, 56]
+  describe '#my_each' do
+    it 'returns the input array' do
+      expect(test_array1.my_each { |x| print x }).to eql(test_array1)
```

Comment:
- [ ] You don't need to print the elements, please erase "print" since it is considered a debugging instruction 
![image](https://user-images.githubusercontent.com/53256139/91753084-f82f0000-eb8c-11ea-940c-81ea85d2f447.png)